packages/web-components/@sfx/products/test/index.test.ts

```diff
@@ -0,0 +1,97 @@
+import { TemplateResult } from 'lit-element';
+import { expect, sinon, spy, stub } from './utils';
+import Products from '../src/products';
+
+describe('Products Component ', () => {
+  let component;
+  beforeEach(() => {
+    component = new Products();
+  });
+
+  describe('general', () => {
```

Comment:
I would call this "properties" or whatever we ended up calling it in the other ones.